### DISCUSSION TITLE
What CPU wattage do you guys use on Silent Mode/on battery? (2024 G14)

### DISCUSSION THREAD

**Discussion by ethanic17** [seerge/g-helper#3066] [General]
Was just wondering what's the best CPU power limit on battery for good battery life while also not being super slow doing normal tasks (web browsing, coding, YouTube, etc.). I already have CPU boost disabled and leave it on Silent & Eco mode but haven't really fiddled around with the wattage sliders/power limits. Currently using 2024 G14 (4070 version).

**Comment by seerge** [comment]
@ethanic17 when you are just browsing / doing "office" work your CPU usually uses a very low amounts of power often couple of watts, so it literally doesn't matter if you set a limit to 20W or to 40W as it doesn't hit that limit anyway. 

You can be just fine by not setting any custom limits i.e. not clicking `Apply` at all. As I think defaults for `Silent` is 50W.

The most impacting thing for battery usage is actually to turn Battery Saver in windows itself (as it shuts down some background tasks) + make sure that you don't have any strange apps or processes running and actually consuming power.

**Reply by ethanic17** [reply]
Ahh okay, the only slight annoyance I have with Windows Battery Saver is that it automatically sets Microsoft Edge's power saving profile to "Maximum savings" and the refresh rate looks choppy (I have 120Hz enabled all the time as I notice it doesn't impact battery life that much with what I do) with noticeable slowdowns. 

My "base" discharge rate is around 5W doing absolutely nothing (although with applications open), and when I'm browsing, I usually see my discharge go up to around 10W, sometimes peaking around 14W. My slash lighting & keyboard backlight are disabled and my screen brightness is around 60/70%. It's not horrible battery life by any means but was just wondering how people are able to get 10+ hours normally whereas I usually get around 5-6 hours.

**Reply by seerge** [reply]
You can change that in Edge settings so it won't be choppy.

To get 10 hours you need discharge of 7-8W (assuming you have ~70Wh+ battery) :) You are not that far.

If you watch youtube for example, what I have noticed that you need to watch it full screen in edge / chrome , then it seem to use less power by some reasons.

**Reply by ethanic17** [reply]
I have 93% battery health so about 67Wh of battery runtime; to achieve 10 hours I would need to average around 7 watts of discharge, but I'm not sure if that would be realistic lol. Since I'm already at 5W doing nothing (with programs open though), there would only be around an additional 2W to the CPU for any immediate tasks which I feel like would slow down things a lot 🥲.

I also wasn't 100% sure how fPPT worked as I know it's a 2-second boost which would help make things snappier even if I limited the sustained wattage to like 5W ish, but after those two seconds; how long would it be until the CPU would be able to use the fPPT wattage again?

And I've also noticed that as well, for some reason it uses less power when in fullscreen mode. But the content itself would also play a factor (bc of the oled panel), or would it not really impact much? And for Edge, would I just disable efficiency mode altogether?

**Reply by seerge** [reply]
@ethanic17 5W is a sign that everything is ok with your laptop. Meaning no random processes running :)

As far as I remember fPPT / sPPT usually resets when there is no load, like in 2 minutes or so. Honestly i prefer to set same limit for all 3 sliders (if I do) to keep things easy and simple.

In Edge - you can just disable efficiency mode completely.

**Reply by ethanic17** [reply]
Ahh okay, thanks! I haven't tested it out, but does disabling Efficiency mode impact battery life quite a lot?

**Reply by seerge** [reply]
@ethanic17 it does impact a little. Main benefit of Battery Saver in windows - it stops random tasks that windows like to run (like updates, some internal maintenance, etc)

**Reply by ethanic17** [reply]
Gotcha, thanks again!

**Comment by 0miicr0n** [comment]
A large part of battery drain I've found in my 2023 G14 is the iGPU. For whatever reason Ghelper doesn't seem to limit iGPU power, though I imagine that varies between different models. If you care enough you can install a software called HWinfo that will tell you what's consuming power on your computer.

**Reply by seerge** [reply]
You can't limit iGPU separately :) But iGPU is part of your APU (CPU + iGPU). So SPL / sPPT / fPPT limits affect that.

**Reply by 0miicr0n** [reply]
Really? Strange, HWinfo says iGPU sits between 4-8W when I have my CPU limited to 8W on the SPL and sPPT. Just browsing causes it to go over my TPP limit. Wonder what could be causing that, but thanks for the info!

**Reply by seerge** [reply]
From my experience HWinfo often adds CPU power to iGPU (especially in case of AMD)

**Reply by 0miicr0n** [reply]
Could certainly be the case I suppose, it says it is accounting for total rail power. Thank you for clearing that up for me! Also, I saw a that you posted a while ago that you weren't planning on adding any more performance/power management features to GHelper because people were accusing you of damaging their batteries or something silly like that. Was just wondering if you still don't have any plans to add more performance controls?

**Reply by seerge** [reply]
I simply can't add any power management, as G-Helper is not managing power in a first place :) Power is managed by firmware / BIOS.

App just gives you access to existing settings on your device via same endpoints that AC uses. If you had power sliders in AC - you will see them in G-Helper, as they are same power sliders.

**Reply by 0miicr0n** [reply]
Ah, I see! Makes sense now that you point that out. Thank you again!
